Formula/boost-python.rb

```diff
@@ -78,12 +58,19 @@ def install
         boost::python::def("greet", greet);
       }
     EOS
-    Language::Python.each_python(build) do |python, _|
-      pyflags = (`#{python}-config --includes`.strip + " " +
-                 `#{python}-config --ldflags`.strip).split(" ")
-      system ENV.cxx, "-shared", "hello.cpp", "-L#{lib}", "-lboost_#{python}", "-o", "hello.so", *pyflags
-      output = `#{python} -c "from __future__ import print_function; import hello; print(hello.greet())"`
-      assert_match "Hello, world!", output
-    end
+
+    pyincludes = Utils.popen_read("python-config --includes").chomp.split(" ")
+    pylib = Utils.popen_read("python-config --ldflags").chomp.split(" ")
+
+    system ENV.cxx, "-shared", "hello.cpp", "-L#{lib}", "-lboost_python", "-o",
+           "hello.so", *pyincludes, *pylib
+
+    output = <<~EOS
+      from __future__ import print_function
+      import hello
+      print(hello.greet())
+    EOS
+
```

Comment:
Let's not bother with the empty line here.

Reply:
Done